packages/sdk/src/types/runtime/behaviors/triggeredAction.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ActionHandler, SetAudioStateOptions } from '../../..';
+
+/**
+ * Describes and action handler with an associated triggered action.
+ */
+export type ActionHandlerWithTriggeredAction = {
+	handler?: ActionHandler;
+	triggeredAction?: TriggeredAction;
+};
+
+/**
+ * Plays an animation in response to the associated event when fired.
+ */
+export type PlayAnimationTriggeredAction = {
+	type: 'play-animation',
+	/**
+	 * The name of the animation to play.
+	 */
+	animationName: string,
+	/**
+	 * The actor on which to play the animation. If omitted, the animation plays on the actor associated with
+	 * the event that fired.
+	 */
+	targetId?: string
+};
+
+/**
+ * Stops an animation in response to the associated event when fired.
+ */
+export type StopAnimationTriggeredAction = {
+	type: 'stop-animation',
+	/**
+	 * The name of the animation to stop.
+	 */
+	animationName: string,
+	/**
+	 * The actor on which to stop the animation. If omitted, the animation plays on the actor associated with
+	 * the event that fired.
+	 */
+	targetId?: string
+};
```

Comment:
Love triggeredactions for sounds and effects, and anything that generally can't mess up synchronization. 
I think we could allow animations on single-user actors as well, but I am afraid that allowing triggeraction animations on regular actors can cause too desyncs if developers don't watch out. And in general we try to protect developers from "be careful with this" kind of feature, so I'd rather restrict it to known safe places.

I'd think that once we refactor animation to run similar to mediainstances, we would be able to mark a subset of animations (the ones that animate parameters such as color etc.) to be safe, whereas we can block changing anything that affects the physics simulation (transform, enable/disable, collision, rigid body parameters).